.github/workflows/publish.yaml

```diff
@@ -0,0 +1,30 @@
+# This workflow will upload a Python Package using Twine when a release is created
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+  
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWRD }}
```

Comment:
This should be `PYPI_PASSWORD`, right?

Reply:
you can name it as you like in your GitHub secrets, but yes in principle it's a typo! :)